src/www/system_advanced_firewall.php

```diff
@@ -732,19 +728,31 @@
                   </div>
                 </td>
               </tr>
-              <tr>
-                <td><a id="help_for_noantilockout" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Disable anti-lockout"); ?></td>
+              <tr id="antilockout_interface">
```

Comment:
To be frank adding a better way but removing the old way to make it "safe" is questionable at best.